api/Window.json

```diff
@@ -9090,17 +9092,12 @@
             "edge": {
               "version_added": false
             },
-            "firefox": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": "20",
-                "notes": "The mimimal size is clamped to prevent the window from being too small for the user to interact with."
-              }
-            ],
+            "firefox": {
+              "version_added": "1",
+              "notes": "Since Firefox 20, the mimimal size is clamped to prevent the window from being too small for the user to interact with."
```

Comment:
Since this note won't be important for anyone to know about, I'd prefer for it to be on a 1-20 range. Maybe "The mimimal size is not clamped, so the window can be too small for the user to interact with.".

But documenting this seems to have been low value even at the time, or how would one depend on the non-clamping behavior in a useful way?

I'd prefer to remove the note, but once again calling for the advice of @ddbeck and @Elchi3.

Reply:
After a few quick searches, I'm skeptical that this was ever implemented anywhere other than Firefox. Since it's non-standard and (apparently) Firefox-only, there's no real compatibility issue to account for here. If you want to remove the note, I'd probably copy this text into the MDN page first, then remove it. I don't feel strongly about it though, so I'm open to other ideas.

Reply:
I agree with Daniel. This should live in the docs.

Reply:
I gave it a try, can someone take a look? https://wiki.developer.mozilla.org/en-US/docs/Web/API/Window/sizeToContent

Reply:
LGTM

Reply:
@vinyldarkscratch can you simplify this PR to just `"version_added": "1"` for Firefox? With that change, this should be good to go!